src/test/mir-opt/const_allocation2/i686/rustc.main.ConstProp.after.mir

```diff
@@ -0,0 +1,67 @@
+// MIR for `main`
+// source = MirSource { instance: Item(DefId(0:3 ~ const_allocation2[317d]::main[0])), promoted: None }
+// pass_name = ConstProp
+// disambiguator = after
+
+fn  main() -> () {
+    let mut _0: ();                      // return place in scope 0 at $DIR/const_allocation2.rs:2:11: 2:11
+    let _1: &[(std::option::Option<i32>, &[&u8])]; // in scope 0 at $DIR/const_allocation2.rs:3:5: 3:8
+    let mut _2: &&[(std::option::Option<i32>, &[&u8])]; // in scope 0 at $DIR/const_allocation2.rs:3:5: 3:8
+
+    bb0: {
+        StorageLive(_1);                 // bb0[0]: scope 0 at $DIR/const_allocation2.rs:3:5: 3:8
+        StorageLive(_2);                 // bb0[1]: scope 0 at $DIR/const_allocation2.rs:3:5: 3:8
+        _2 = const Scalar(alloc0+0): &&[(std::option::Option<i32>, &[&u8])]; // bb0[2]: scope 0 at $DIR/const_allocation2.rs:3:5: 3:8
+                                         // ty::Const
+                                         // + ty: &&[(std::option::Option<i32>, &[&u8])]
+                                         // + val: Value(Scalar(alloc0+0))
+                                         // mir::Constant
+                                         // + span: $DIR/const_allocation2.rs:3:5: 3:8
+                                         // + literal: Const { ty: &&[(std::option::Option<i32>, &[&u8])], val: Value(Scalar(alloc0+0)) }
+        _1 = (*_2);                      // bb0[3]: scope 0 at $DIR/const_allocation2.rs:3:5: 3:8
+        StorageDead(_2);                 // bb0[4]: scope 0 at $DIR/const_allocation2.rs:3:8: 3:9
+        StorageDead(_1);                 // bb0[5]: scope 0 at $DIR/const_allocation2.rs:3:8: 3:9
+        _0 = ();                         // bb0[6]: scope 0 at $DIR/const_allocation2.rs:2:11: 4:2
+        return;                          // bb0[7]: scope 0 at $DIR/const_allocation2.rs:4:2: 4:2
+    }
+}
+
+alloc0 (static: FOO, size: 8, align: 4) {
+    ╾alloc24+0╼ 03 00 00 00
+}
+
+alloc24 (size: 48, align: 4) {
+    0x00 │ 00 00 00 00 __ __ __ __ ╾alloc9+0─╼ 00 00 00 00
+    0x10 │ 00 00 00 00 __ __ __ __ ╾alloc14+0╼ 02 00 00 00
+    0x20 │ 01 00 00 00 2a 00 00 00 ╾alloc22+0╼ 03 00 00 00
```

Comment:
Could you add/change a testcase that does not fit any more (e.g. by also having a 2-digit offset)?